server/src/utility/scheduler.js

```diff
@@ -0,0 +1,38 @@
+const schedule = require('node-schedule');
```

Comment:
Does schedule persist all scheduled jobs in a DB/file, or is it just in memory?
We need to make sure that it persists

Reply:
In the documentation provided for the `node-schedule` package, it is given that jobs persist as long as the script is running.
> Note that Node Schedule is designed for in-process scheduling, i.e. scheduled jobs will only fire as long as your script is running, and the schedule will disappear when execution completes. If you need to schedule jobs that will persist even when your script isn't running, consider using actual cron.
> Documentation

Reply:
Okay.
But we need it to persist even after the script finishes/restarts. (Example scenario: server crashes / server is updated)

Reply:
For this, we can go for using mongodb, redis or static files.

Reply:
Yes. But we will also have to read and schedule it again (after server starts), if the event does not exist.

Lets use MongoDB. We can refer to events by id, and schedule it again, if such an event does not exist.

Considering this is a big change, lets raise an issue for it, and do it in another PR. We can merge this PR once the other points are addressed

Reply:
We can do something like an user will contain an event id field for which he/she has added the reminder for.

Whenever the server restarts we can schedule the jobs using that field.